improve-digital-htb-validator.js

```diff
@@ -19,6 +19,34 @@
 
 var Inspector = require('../../../libs/external/schema-inspector.js');
 
+var custom = {
+    impliesPresence: function (schema, candidate) {
+        var implication = schema.$impliesPresence;
+        var impliedIf = implication.if;
+        var impliedThen = implication.then;
+        if (candidate[impliedIf] && typeof candidate[impliedThen] === 'undefined') {
+            this.report('if ' + impliedIf + 'exists then ' + impliedThen + ' must also exist');
+        }
+    }
+};
+Inspector.Validation.extend(custom);
+
+custom = {
+    mustIncludeOneOnly: function (schema, candidate) {
+        var mustIncludeOneOnly = schema.$mustIncludeOneOnly.params;
+        var occurrenceCounter = 0;
+        for (var index = 0; index < mustIncludeOneOnly.length; index++) {
```

Comment:
When running the following:
```
let config = {
    "xSlots": {
        "1": {
            "placementId": 1053688
        },
        "2": {
            "publisherId": 1032,
            "placementKey": "data_team_test_hb_smoke_test"
        }
    },
    "mapping": {
        "slot1": [
            "1"
        ],
        "slot2": [
            "2"
        ]
    },
    "accountId": "8263",
    "timeout": 1000   
}
partnerValidator(config)
```
An error was thrown: 
```
TypeError: Cannot read property 'length' of undefined
```
at line 38.

Please make corresponding changes.

Reply:
I have fixed this now.  There was a typo.